odxtools/communicationparameter.py

```diff
@@ -14,41 +38,39 @@ def __init__(self,
                  description: Optional[str] = None,
                  protocol_sn_ref: Optional[str] = None,
                  prot_stack_sn_ref: Optional[str] = None):
-        self.value = value
+        self._value = value
         self.id_ref = id_ref
         self.description = description
         self.protocol_sn_ref = protocol_sn_ref
         self.prot_stack_sn_ref = prot_stack_sn_ref
+        self.comparam: Optional[BaseComparam] = None
 
     def __repr__(self) -> str:
         val = self.value
-        if not isinstance(self.value, list):
+        if isinstance(val, str):
             val = f"'{val}'"
-
-        return f"CommunicationParameter('{self.id_ref}', value={val})"
+        return f"CommunicationParameter('{self.short_name}', value={val})"
 
     def __str__(self) -> str:
-        val = self.value
-        if not isinstance(self.value, list):
-            val = f"'{val}'"
+        return self.__repr__()
 
-        return f"CommunicationParameter('{self.id_ref}', value={val})"
+    def _resolve_references(self, odxlinks: OdxLinkDatabase):
+        # Temporary lenient until tests are updated
+        self.comparam = odxlinks.resolve_lenient(self.id_ref)
+        if not self.comparam:
+            warnings.warn(f"Could not resolve COMPARAM '{self.id_ref}'", OdxWarning)
```

Comment:
@andlaus this fails for the `somersaultecu` due to missing com param, but works fine for other PDX files

Reply:
I have a fix for that: https://github.com/kayoub5/odxtools/pull/1 (this is a PR against this PR, if you agree, please merge.)